reana/reana_benchmark/cli.py

```diff
@@ -165,6 +201,14 @@ def launch(
 @reana_benchmark.command(name="analyze")
 @workflow_option
 @workflow_range_option
+@click.option(
+    "--datetime",
+    "datetime_range",
+    help="Datetime range for the execution progress plot, "
```

Comment:
Minor: I think we could specify a bit more what this option does, something like `Filter execution progress plot data by a specific datetime range `

Reply:
Agree that a better description is needed. But instead of `filter ... data`, I would just put `Filter execution progress plot to contain data within the specified datetime range`.

Reply:
Done